pkg/cloudprovider/providers/azure/azure_standard_test.go

```diff
@@ -157,10 +158,120 @@ func TestMapLoadBalancerNameToVMSet(t *testing.T) {
 			clusterName:   "azure",
 			expectedVMSet: "azuretest",
 		},
+		{
+			description:   "default standard external LB should map to empty string",
+			lbName:        "azure",
+			useStandardLB: true,
+			clusterName:   "azure",
+			expectedVMSet: "",
+		},
+		{
+			description:   "default standard internal LB should map to empty string",
+			lbName:        "azure-internal",
+			useStandardLB: true,
+			clusterName:   "azure",
+			expectedVMSet: "",
+		},
+		{
+			description:   "non-default standard external LB should map to empty string",
+			lbName:        "azuretest-internal",
+			useStandardLB: true,
+			clusterName:   "azure",
+			expectedVMSet: "",
+		},
+		{
+			description:   "non-default standard internal LB should map to empty string",
+			lbName:        "azuretest-internal",
+			useStandardLB: true,
+			clusterName:   "azure",
+			expectedVMSet: "",
+		},
 	}
 
 	for _, c := range cases {
+		if c.useStandardLB {
+			az.Config.LoadBalancerSku = loadBalancerSkuStandard
+		}
```

Comment:
should set LoadBalancerSku back for !useStandardLB. This works now since the Standard SKU cases are in the end.